src/commands.ts

```diff
@@ -620,33 +621,37 @@ export async function getEphemeralPrivKey(index: string) {
     this.resume();
 }
 
-export async function enableDelegatedProver() {
+export async function setProverMode(mode: ProverMode) {
     this.pause();
-    this.account.setDelegatedProverEnabled(true);
-    this.echo(`Delegated prover enabled`);
-    this.resume();
-}
-
-export async function disableDelegatedProver() {
-    this.pause();
-    this.account.setDelegatedProverEnabled(false);
-    this.echo(`Delegated prover disabled`);
+    this.account.setProverMode(mode);
```

Comment:
Is it need to check the input value here?

Reply:
I guess no, input value will be checked in setProverMode in zkbob-client-js. https://github.com/zkBob/zkbob-client-js/blob/cf0b9cb25ba1d3487d6698ea667fb7042b92695e/src/client.ts#L415

Reply:
Yes, agreed with that.

Another proposal here. Maybe it's make sense to perform case-insensitive comparing (here or in the client library)? Or make aliases for the modes (0 = 'Delegated', 1 =..., 2 =...'). Or implement autosubstitution in the CLI. Using this command is very inconvenient currently due to long mode string typing. Wdyt?

Reply:
Wdyt?

Reply:
I think commands like `set-prover-mode delegatedwithfallback` or `set-prover-mode 0` are not very clear.

Reply:
The typing process with autocomplete become more convenient